resources/scripts/components/server/Console.tsx

```diff
@@ -201,8 +201,12 @@ export default () => {
         };
     }, [ connected, instance ]);
 
+    const openConsole = () => {
+        window.open(window.location.href + '/console', 'Server Console', 'height=1000,width=1500');
```

Comment:
Can you do a smaller window size, this is massive.

Pick whatever seems best to you:
a.) 640x whatever the other dimension that normally goes with this is.
b.) Whatever the old console size was in 0.7
c.) Some scaling based on the detected resolution of the user. Like, 1/4 of the scale, or 1/3.

Reply:
640x480 should look perfectly